boot/freeldr/freeldr/arch/i386/i386bug.c

```diff
@@ -118,7 +118,10 @@ i386PrintExceptionText(ULONG TrapIndex, PKTRAP_FRAME TrapFrame, PKSPECIAL_REGIST
 
     PrintText("FreeLdr " KERNEL_VERSION_STR " " KERNEL_VERSION_BUILD_STR "\n"
               "Report this error on the ReactOS Bug Tracker: https://jira.reactos.org\n\n"
-              "0x%02lx: %s\n\n", TrapIndex, i386ExceptionDescriptionText[TrapIndex]);
+              "0x%02lx: Exception %02X: %s\n\n",
+              TrapIndex,
+              TrapIndex,
+              i386ExceptionDescriptionText[TrapIndex]);
```

Comment:
It should be fine to keep these 3 guys on the same line
```suggestion
              TrapIndex, TrapIndex, i386ExceptionDescriptionText[TrapIndex]);
```